### ISSUE TITLE
Fallback experiment is not always inserted into entry-point experiment

### ISSUE THREAD

**Issue by smathot** [open-cogsci/omm-client#9]
@nclaidiere reported that, when specifying a fallback experiment in the OMM extension, the path is not inserted into the entry-point experiment; that is, when you look at the *announce* item in the entry point, the 'Fallback experiment' field is blank, even though it was specified in the extension.

I haven't been able to reproduce this myself. @nclaidiere can you reproduce this yourself, and if so, how?

**Comment by nclaidiere** [comment]
Hi, so after further testing I think the problem is with admin privileges. We tried this morning to install a new server with a new ip address, so I went to all the clients and changed the IP on the OMM plugin. However, the software continued to query the old one. Looking at the entry point, the old IP was still there, not the new one. I therefore changed the entry point and saved it in the correct folder. I received an error message saying that I did not have permission to write to that folder. I restarted opensesame in administrator mode, I then could save it and the change became effective.
So I think the OMM plugin is trying to save the data and without admin privileges that is not possible, could that be it?
